lib/modules/newCommentCount.js

```diff
@@ -336,6 +341,9 @@ addModule('newCommentCount', function(module, moduleID) {
 	}
 
 	function handleButton(threadid, action) {
+		if (module.options.showSubscribeButton.value == true){
```

Comment:
Also, I don't remember what javascript's default return value is, but I think this would break L170-L179 since I doubt Jquery's `.append` supports it. May want to do the boolean check there (and come to think of it, the check is incorrect, it should be `!module.options.showSubscribeButton.value`, since you want it to not had the button if it's false)